format/ecs.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-param-reassign */
+const { format } = require('logform');
+
+/*
+ * function ecs (info)
+ * Aligns the base info object fields to the ECS schema
+ */
+module.exports = format((info) => {
+  if (info.timestamp) {
+    info['@timestamp'] = info.timestamp;
+    delete info.timestamp;
+  }
+
+  if (info.level) {
+    info.log = { level: info.level, ...info.log };
+    delete info.level;
+  }
+
+  if (info.stack) {
+    info.error = { stack_trace: info.stack, ...info.error };
```

Comment:
Ecs format uses snake case?

Reply:
This is the field from ECS. I dont know if entire ECS is snake case.